src/My.Extensions.Localization.Json/My.Extensions.Localization.Json.csproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="2.0.0" />
```

Comment:
It works great, but I dislike this line, because it brings dependency on MVC which I think it's not suitable, coz all of the data annotations stuff is MVC related

I think the better option is create a new project that has the `AddDataAnnotationsJsonLocalization`, in this way you can reference both assemblies from within the sample

Reply:
Perhaps I need to create a new package such as https://github.com/hishamco/My.AspNetCore.Mvc.TagHelpers

Reply:
Or even better to add it to this repo https://github.com/hishamco/My.AspNetCore.Localization.Json while it's related

Reply:
@Sandeep321 seems we took your consideration https://github.com/hishamco/My.Extensions.Localization.Json/issues/2#issuecomment-442398221 in real

thanks

Reply:
So do you think I should close this PR and open a new one for https://github.com/hishamco/My.AspNetCore.Mvc.TagHelpers? 
The trade-off here is that we will need to reference 2 NuGet packages for localization.

Reply:
> The trade-off here, I think, is that we will need to reference 2 NuGet packages for localization.

Make it as it is will bring extra MVC packages that you don't need unless you are using MVC web app, FYI till now some of localization stuff are belong to MVC assembly not localization one

So don't close the PR, let's think about the right way to support this

Reply:
What if I told you there're more the 3 or 4 localization packages
`Microsoft.Extensions.Localization.Abstractions`, `Microsoft.Extensions.Localization`, `Microsoft.AspNetCore.Localization`, `Microsoft.AspNetCore. Mvc.Localization`.. etc

Reply:
FYI referencing the JsonMvcLocalizer package that I'm suggesting is the only thing that you need to reference and it will pull all the dependencies that is required